hydra/test_utils/test_utils.py

```diff
@@ -423,3 +424,25 @@ def assert_text_same(
         print(diff)
         print("-------------------------------")
         assert False, "Mismatch between expected and actual text"
+
+
+def assert_regex_match(
```

Comment:
can you explain this function?
I am not sure I understand it and I suspect it doesn't really work in various cases.